index.bs

```diff
@@ -349,6 +349,218 @@ One example of a feature that should be limited to secure contexts
+which remain a major use of web technology.
+
+If non-visual consumers of a page (such as search engines or screenreaders)
+seem likely to need the information (at least for well-designed pages),
+then it probably belongs in the markup rather than in the CSS.
+
+<p class="example">For example, information about the human language (English, Chinese, Arabic, etc.)
+that the text is in belongs in the markup
+(in particular, the <code><a href="https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes">lang</a></code> attribute),
+even though a number of the uses of that information (such as font selection or hyphenation)
+are related to the presentation.
+On the other hand, information about the font size makes sense to be in the style,
+since in pages that use HTML correctly
+the information in font sizes should also be obtainable from the markup
+(e.g., <code highlight="html">&lt;h1&gt;</code>, <code highlight="html">&lt;strong&gt;</code>)
+even though on some poorly designed pages the notion of what is a heading might not be visible to search engines or screenreaders except through the style
+(e.g., <code highlight="html">&lt;div style="font-size: 1.8em"&gt;</code>).
+
+<h3 id="css-property-separation">Separate CSS properties based on what should cascade separately</h3>
+
+Decisions about what should be in the same CSS property or in different CSS properties
+should be based on what makes sense to set independently.
+CSS cascading allows declarations (which are property-value pairs)
+from different rules or different style sheets
+to override one another in a process called the cascade.
+Thus things that should always be set together belong in the same properties,
+but things that make sense to set in different places should be in separate properties.
+
+<p class="example">For example, the "size" and "sink" aspects of the 'initial-letters' property
+belong in a single property
+because they are part of a single initial letter effect
+(e.g., a drop cap, sunken cap, or raised cap)
+and should nearly always be specified together.
+However, the 'initial-letters-align' property should be separate because it
+sets an alignment policy for all of these effects across the document
+which is a general stylistic choice
+and a function of the script (e.g., Latin, Cyrillic, Arabic) used in the document.
+
+<h3 id="css-inherited-or-not">Make appropriate choices for whether CSS properties are inherited</h3>
+
+An important choice in the design of every CSS property is whether the property is
+<a href="https://drafts.csswg.org/css-cascade-3/#inherited-property">inherited</a>.
+This affects what happens when no value is specified for the property on an element.
+If the property is not inherited, then that element gets the property's initial value.
+If the property is inherited, then that element gets the parent element's computed value.
+
+The decision about whether a property should be inherited should be based largely
```

Comment:
I think it should be noted that, to a first approximation, inherited or not is about whether the property affects text or boxes. The number of exceptions to that rule is fairly small, and you can go into more detail about those exceptions afterwards.

Reply:
If that's added I think it should go at the end of the section, as I'm not sure there's a way of stating that rule that provides as much explanation as what's currently here.

Reply:
I've added a paragraph on this as the third paragraph of the section.